src/table/utils/handle-key-press.js

```diff
@@ -134,10 +134,17 @@ export const bodyHandleKeyPress = ({
   selectionsAPI = null,
 }) => {
   const isInSelectionMode = selectionsAPI?.isModal();
+  const isMultiValueSelections = () =>
```

Comment:
Do you expect to use this anywhere else? If not, no need to defined a function

Reply:
or if you break it out so it is testable + reduces complecxity of `bodyHandleKeyPress`

Reply:
Updated